### PR TITLE
Instructor notes

### LINKED ISSUE #476: Restructure instructor notes
The current version of [instructor notes](https://datacarpentry.org/r-socialsci/instructor/instructor-notes.html) has a lot of redundant information on setup as well as potentially unnecessary information. 

Additionally consider providing guidance on example lesson plans (example: https://swcarpentry.github.io/r-novice-gapminder/instructor/instructor-notes.html)

### PR DESCRIPTION
**Pull request by juanfung**
Closes #476 

Adds 

- Suggested lesson plans to instructor notes
- Summary of "Introduction to R" episode in the Narrative section
- "Data" section moved to beginning of Narrative

FYI: Accidentally cited #477 in commits. 

<details>

For more guidance on how to contribute changes to a Carpentries project, please review [the Contributing Guide](CONTRIBUTING.md) and [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html).

Please keep in mind that lesson Maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

</details>

### CODE REVIEW

**Comment by juanfung** [comment]
@jessesadler 🚀

File: index.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ less time, and with less pain. The lessons below were designed for
 those interested in working with social sciences data in R.
 
 This is an introduction to R designed for participants with no
-programming experience. These lessons can be taught in a day (~ 6
-hours). They start with some basic information about R syntax, the
+programming experience. These lessons can be taught in a half-day,
+full-day, or over a two-day workshop (see
+[Instructor Notes](https://datacarpentry.org/r-socialsci/instructor/instructor-notes.html)).
```
Comment (jessesadler) [inline index.md:13]: The link to the instructor notes is put in twice, once in the markdown link and once outside it. Delete the second instance.

**Review by jessesadler — COMMENTED** [review]
I noted one change to make in the index.md file where a url is written twice. Otherwise the content looks good. However, I was looking over the Workbench material and figured out why the Instructor notes page has headings for all of the lessons at the bottom of the page. It looks like the preferred method is to put episode specific notes in that episode within an instructor div. See here https://carpentries.github.io/workbench/transition-guide.html#instructor-notes. I think it would be a good idea to do this. Just moving what we have here to the individual episodes at the top of the document. A benefit of this is that within the instructor view the instructor notes are visible within the episode instead of on an external page. Let me know what you think @juanfung. I am also happy to do it, if you approve, if you do not have the time right now.

**Comment by juanfung** [comment]
@jessesadler great catch! Also, thank you for looking into the episode heading thing.

**Comment by juanfung** [comment]
@juanfung Reminder

- [ ] Move narrative episode summaries from instructor notes to each respective episode

**Review by jessesadler — APPROVED** [review]
Looks good!

**Pull request merged**